types/validator_set.go

```diff
@@ -749,11 +749,15 @@ func (vals *ValidatorSet) VerifyFutureCommit(newSet *ValidatorSet, chainID strin
 
 // VerifyCommitTrusting verifies that trustLevel ([1/3, 1]) of the validator
 // set signed this commit.
+//
 // NOTE the given validators do not necessarily correspond to the validator set
 // for this commit, but there may be some intersection.
+//
+// Panics if trustLevel is invalid.
```

Comment:
does this actually panic? it looks like it returns an error to me

Reply:
It does. Line 763 - `panic(fmt.Sprintf("trustLevel must be within [1/3, 1], given %v", trustLevel))`